tmt-app/public/index.html

```diff
@@ -3,6 +3,10 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
```

Comment:
FYI there is an npm package for the popping font, we can ship it as a dependency rather than a cdn

Reply:
Is there a specific npm package or would this one be fine [https://www.npmjs.com/package/@fontsource/poppins](url)

Reply:
I believe this one should be fined